source/includes/_feature-importing.md

```diff
@@ -31,7 +31,21 @@ ds <- newDatasetFromFile("my.csv", name="my survey")
 # All three steps are handled within newDatasetFromFile
 ```
 
-POST a Dataset Entity to the datasets catalog.
+POST a Dataset Entity to the datasets catalog. The available initial attributes
+to create a new dataset entity are (all optional):
+
+
+Name | Type | Description
+---- | ---- | -----------
+name | string | Human-friendly string identifier.
+description | string | Optional longer string.
+archived | boolean | Whether the variable should be hidden from most views; default: false.
+owner | Url | The URL of the owner of the dataset, if omitted will use the logged user as owner. 
```

Comment:
It allows to create datasets for teams. If a user url is used it has to be you, it will err otherwise.